### PR TITLE
feat(hr): Remove cnomua

### PR DESCRIPTION
**Pull request by dr1rrb**
linked https://github.com/unoplatform/uno/issues/15918

## Feature
Remove the `CreateNewOnMetadataUpdateAttribute` in code gen

## What is the current behavior?
All `UserControl` and `ResourceDictionnary` are flagged with the `CreateNewOnMetadataUpdateAttribute`

## What is the new behavior?
We rely in HR patches

## PR Checklist
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UI.Tests/Uno.UI.Unit.Tests.csproj [hunk 1]
```diff
@@ -12,11 +12,10 @@
 	</PropertyGroup>
 	<!--
 	Uncomment to troubleshoot source generation
+	-->
 	<PropertyGroup>
-		<UnoUISourceGeneratorDebuggerBreak>True</UnoUISourceGeneratorDebuggerBreak>
 		<EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
 	</PropertyGroup>
-	-->
```
Comment (jeromelaban) [inline src/Uno.UI.Tests/Uno.UI.Unit.Tests.csproj:19]: keep the original

File: src/Uno.UI/Uno.UI.Tests.csproj [hunk 2]
```diff
@@ -6,6 +6,9 @@
 	</PropertyGroup>
 
 	<Import Project="../targetframework-override.props" />
+	<PropertyGroup>
+		<EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
```
Comment (jeromelaban) [inline src/Uno.UI/Uno.UI.Tests.csproj:10]: Not sure if it's useful by default.

**Review by jeromelaban — APPROVED** [review]

File: src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/SOSLIOFPLR/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs [hunk 3]
```diff
@@ -267,8 +246,36 @@ public _View Build(object __ResourceOwner_1)
...
+
+							private void ApplyMethod_12(global::Microsoft.UI.Xaml.Controls.StackPanel __p1, _MainPage_d6cd66944958ced0c513e0a04797b51d_TestReproMainPageSC0 __that, global::Microsoft.UI.Xaml.NameScope __nameScope)
+				{
+					global::Uno.UI.FrameworkElementHelper.SetBaseUri(__p1, __baseUri_MainPage_d6cd66944958ced0c513e0a04797b51d, "file:///C:/Project/0/MainPage.xaml", 14, 8);
+					__p1.CreationComplete();
+				}
+
 		}
-		private global::Microsoft.UI.Xaml.Markup.ComponentHolder _component_0_Holder { get; } = new global::Microsoft.UI.Xaml.Markup.ComponentHolder(isWeak: true);
+		private global::Microsoft.UI.Xaml.Markup.ComponentHolder _component_0_Holder { get; set; } = new global::Microsoft.UI.Xaml.Markup.ComponentHolder(isWeak: true);
```
[the remark this answers is not in the document]
Comment (dr1rrb) [reply src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/SOSLIOFPLR/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs]: Removed where it was easy to remove. Anyway in release this is a field that is also get/set

File: src/Uno.UI/Uno.UI.Tests.csproj (hunk 2 above)
Comment (dr1rrb) [reply src/Uno.UI/Uno.UI.Tests.csproj]: Removed

File: src/Uno.UI.Tests/Uno.UI.Unit.Tests.csproj (hunk 1 above)
Comment (dr1rrb) [reply src/Uno.UI.Tests/Uno.UI.Unit.Tests.csproj]: Removed

File: src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/SBUIIOFFE/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs [hunk 4]
```diff
@@ -34,24 +34,15 @@
...
 		private const string __baseUri_MainPage_d6cd66944958ced0c513e0a04797b51d = "ms-appx:///TestProject/";
 		private global::Microsoft.UI.Xaml.NameScope __nameScope = new global::Microsoft.UI.Xaml.NameScope();
 		private void InitializeComponent()
 		{
-			var __resourceLocator = new global::System.Uri("file:///C:/Project/0/MainPage.xaml");
-			if(global::Uno.UI.ApplicationHelper.IsLoadableComponent(__resourceLocator))
-			{
-				global::Microsoft.UI.Xaml.Application.LoadComponent(this, __resourceLocator);
-				return;
-			}
```
[the remark this answers is not in the document]
Comment (dr1rrb) [reply src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/SBUIIOFFE/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs:54]: Will remove XAML and partial reload in a separated PR

File: src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/TMEWSFEL/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs [hunk 5]
```diff
@@ -50,21 +50,21 @@ private void InitializeComponent()
...
-				// Source 0\MainPage.xaml (Line 1:2)
-				
-				// WARNING Property c0.base does not exist on {using:TestRepro}BasePage, the namespace is http://www.w3.org/XML/1998/namespace. This error was considered irrelevant by the XamlFileGenerator
+			// Source 0\MainPage.xaml (Line 1:2)
+			
+			// WARNING Property __p1.base does not exist on {using:TestRepro}BasePage, the namespace is http://www.w3.org/XML/1998/namespace. This error was considered irrelevant by the XamlFileGenerator
 			}
 			))
-			.GenericApply(((c1) => 
+			.MainPage_d6cd66944958ced0c513e0a04797b51d_XamlApply((MainPage_d6cd66944958ced0c513e0a04797b51dXamlApplyExtensions.XamlApplyHandler0)(__p1 =>
```
[the remark this answers is not in the document]
Comment (dr1rrb) [reply src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/TMEWSFEL/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs:60]: This test will be pushed in a sub-sequent PR

**Pull request merged**